### ISSUE TITLE
Image Recognition Training Pipeline

### ISSUE THREAD

**Issue by JacobGlennAyers** [JacobGlennAyers/ECE148_Final_Project#2]
We need a pipeline where we can feed in a couple dozen to a hundred or so images of each of the musicians we want to classify on, I am thinking that we just copy-paste a large amount of the code from the Jetson Nano tutorial that made use of Tensorflow and transfer learning from resnet. Because we don't need any segmentation. We can probably take from the emotions script since that already had multiple classes.

**Comment by JacobGlennAyers** [comment]
Pytorch seems to be easier to work with and this pipeline automatically normalizes jpeg files that have different sizes and performs data augmentation. It also uses some transfer learning from a basic ConvNet:
https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html

**Comment by ShohM2** [comment]
Tutorial from class which might be useful.
[https://github.com/mcdeoliveira/notebooks/blob/master/AutonomousVehicles/NNTutorialWithKeras.ipynb](url)

**Comment by JacobGlennAyers** [comment]
The purpose of this script will be to write out a .pb file to be used by the prediction pipeline.

**Comment by JacobGlennAyers** [comment]
First pass of this is completed based on the pytorch Resnet18 transfer learning pipeline.
I will make a small tweak to the script that dynamically names the model weights so I don't have to worry about overwriting prior pytorch models.
Closing for now.
